chapters/develop-organize/reading/reading/README.md

```diff
@@ -18,8 +18,12 @@ To enforce the guidelines we use [automated linter checks](https://github.com/su
 Besides the mentioned guidelines, the linter implements checks for other minor style conventions (such as how to define lists, where should new lines be placed and others).
 However, for the majority of cases, the above should be sufficient.
 
+The reading content type is stored as a folder named `reading` in the course repository.
+Each topic should contain a `reading` folder that includes one or multiple Markdown files, depending on how the topic may be broken down in independent reading parts.
```

Comment:
Should we give a `tree` example here, it gets a bit confusing?

Reply:
I thought about this, but ended up concluding that it's not necessary. Now, on a full stomach, it seems indeed that tree example would alleviate any confusion between topic and chapter.